### PR TITLE
Dump pickled objects of samples filtered by HMM cost

### PR DESCRIPTION
**Pull request by szhan**
Helps with #188

### CODE REVIEW

File: sc2ts/cli.py [hunk 1]
```diff
@@ -229,9 +235,11 @@ def daily_extend(
             num_threads=num_threads,
             show_progress=not no_progress,
         )
-        for ts, date in ts_iter:
-            output = output_prefix + date + ".ts"
-            add_provenance(ts, output)
+        for ts, excluded_samples, date in ts_iter:
+            output_ts = output_prefix + date + ".ts"
+            add_provenance(ts, output_ts)
+            output_excluded_samples = output_prefix + date + ".pickle"
```
Comment (jeromekelleher) [inline sc2ts/cli.py:241]: Maybe add descriptive ".excluded.pickle" or something so we know what they are?

**Review by jeromekelleher — COMMENTED** [review]
Nice - just a quick suggestion on file naming

**Comment by szhan** [comment]
Should we add some tests to check the contents of the pickled samples in the output files?

File: sc2ts/cli.py (hunk 1 above)
Comment (szhan) [reply sc2ts/cli.py]: I'm going for ".excluded_samples.pickle". A bit long, but pretty clear what it is.

**Comment by jeromekelleher** [comment]
Let's not bother for now, we'll likely change the format a bit as we go

**Pull request merged**